src/index.html

```diff
@@ -1,12 +1,390 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#home"
+          class="header__logo logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+            class="logo__img"
+          >
+        </a>
+        <nav class="header__menu menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a
+                class="menu__link is-active"
+                href="#apple"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a
+                class="menu__link"
+                href="#samsung"
+              >Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="menu__item">
              <a
                class="menu__link"
                href="#samsung"
              >
                Samsung
              </a>
            </li>
```
https://github.com/mate-academy/layout_catalog/pull/2455#discussion_r1047604365

generally: you should fix the same issues everywhere, not just where a comment was written